packages/core/chrome/core-chrome-browser-internal/src/ui/project/header.tsx

```diff
@@ -121,18 +215,11 @@ export const ProjectHeader = ({
               navigateToUrl={application.navigateToUrl}
             />
           </EuiHeaderSectionItem>
-
-          <EuiHeaderSectionItem>
-            <HeaderNavControls navControls$={observables.navControlsRight$} />
-          </EuiHeaderSectionItem>
         </EuiHeaderSection>
       </EuiHeader>
+
       {headerActionMenuMounter.mount && (
         <EuiHeader data-test-subj="kibanaProjectHeaderActionMenu">
-          {/* TODO: This puts a group of nav menu items on the right edge of the screen,
-              but it should be possible for apps customize the layout in a grid and use spacers between items.
-              https://github.com/elastic/kibana/issues/158034 */}
-          <EuiHeaderSection />
```

Comment:
removed this comment since we won't focus on more work here in the short-term, and the design may change once we get to it.